tests/ocp/sriov/internal/ocpsriovconfig/.default.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# OCP SR-IOV default configurations.
```

Comment:
What's the reason  to rename the file? it was default.yaml, now it's .default.yaml